src/hooks/routing/clientSideSmartOrderRouter.ts

```diff
@@ -11,6 +13,11 @@ export const AUTO_ROUTER_SUPPORTED_CHAINS: ChainId[] = Object.values(ChainId).fi
   Number.isInteger(chainId)
 )
 
+export function useAutoRouterSupported(): boolean {
+  const { chainId } = useActiveWeb3React()
+  return Boolean(chainId && AUTO_ROUTER_SUPPORTED_CHAINS.includes(chainId))
```

Comment:
This should be an exported function instead of exporting AUTO_ROUTER_SUPPORTED_CHAINS: `isAutoRouterSupportedChain(chainId?: ChainId): boolean`.